simulator/simple_planning_simulator/README.md

```diff
@@ -62,26 +62,57 @@ The purpose of this simulator is for the integration test of planning and contro
+-1.5,-1.7,-1.610613371,-1.470852729,-1.461088567,-1.404871706,-1.368495381,-1.286898193,-1.238396629,-1.1,-0.9943455959,-0.8298192234,-0.8,-0.78
+-1,-1.3,-1.275235376,-1.103445843,-1.088406302,-1.039321358,-1.02105369,-0.9751710693,-0.8854352235,-0.8210113976,-0.607258157,-0.5174264392,-0.5412350047,-0.5610285835
+-0.8,-0.9590832786,-0.9,-0.8186387583,-0.7447984172,-0.7023011258,-0.6545403693,-0.6320942295,-0.5935860191,-0.55048343,-0.4442914804,-0.3901021846,-0.3919597965,-0.35
+-0.6,-0.7658249026,-0.7066297769,-0.6748573388,-0.6454624801,-0.5800721935,-0.5176519761,-0.5081073889,-0.4988131589,-0.4004919454,-0.3257258364,-0.3018138626,-0.3110086312,-0.3
+-0.4,-0.45,-0.3955078655,-0.45,-0.4420551681,-0.378263005,-0.3483463745,-0.3051486357,-0.2979650452,-0.2632949777,-0.2982640387,-0.2920397922,-0.3068711635,-0.25
+-0.2,-0.236347113,-0.24,-0.2466631395,-0.2166629978,-0.2289237699,-0.2494739762,-0.2681446089,-0.2930905187,-0.24,-0.2246614573,-0.1669471084,-0.177229732,-0.1213663697
+0,0,0,-0.05,-0.05,-0.05,-0.05,-0.08,-0.08,-0.08,-0.08,-0.1,-0.1,-0.1
+0.2,0.16,0.12,0.0208379152,0.02,0,0,-0.05,-0.05,-0.05,-0.05,-0.08,-0.08,-0.08
+0.4,0.3750641135,0.3047092516,0.221616677,0.2471973044,0.2350857222,0.231864649,0.2018410901,0.1562028944,0.1616276318,0.14,0.1,0.05,0.05
+0.6,0.52,0.5193656045,0.5070327645,0.4865393415,0.4329989451,0.3958019322,0.354683087,0.3307829017,0.3291051391,0.3347948202,0.3177862573,0.3439621507,0.34
+0.8,0.8239214061,0.8064813329,0.7847310427,0.6810651129,0.6338313891,0.5583948098,0.5275341787,0.4842021213,0.4326073185,0.4077661307,0.3723376985,0.38,0.3985055042
+1,1.1,1.081666906,1.010666969,0.8781165326,0.762716663,0.6873783468,0.6553170752,0.5790677856,0.5361829552,0.48682048,0.45491258,0.3999384688,0.4
+1.5,1.52,1.501457265,1.381911528,1.257342524,1.139809219,1.033006839,0.9103484248,0.8172015502,0.6687928801,0.6055502227,0.5058744415,0.4088174437,0.41
+2,1.8,1.7975859,1.638490281,1.426616009,1.251194668,1.113771502,0.9618652599,0.8098033929,0.6953401352,0.5896689884,0.51189749,0.4180952453,0.42
+```
+
+![acceleration_map](https://private-user-images.githubusercontent.com/39142679/286099389-97a77043-86af-4384-8612-65b2b81fa360.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExNDA1NDMsIm5iZiI6MTcwMTE0MDI0MywicGF0aCI6Ii8zOTE0MjY3OS8yODYwOTkzODktOTdhNzcwNDMtODZhZi00Mzg0LTg2MTItNjViMmI4MWZhMzYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDAyNTcyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRhMTM1YTMyYzBhNDEzNDA4MTU4MGEyMTdlMDhlNDUzYmMxYTUwOGRjOWJkYmU5NWQyNjg5OGMzNDIyOGNiOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.D7rTJhrKAz0J8FmlYOQ1QNbYjIEpMiPlBq9JHCbu1PY)
```

Comment:
The link is broken

Reply:
oh, can you see https://private-user-images.githubusercontent.com/39142679/286107139-c41d714a-9ffc-4601-9a62-e2427dca014f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExNDc1NzMsIm5iZiI6MTcwMTE0NzI3MywicGF0aCI6Ii8zOTE0MjY3OS8yODYxMDcxMzktYzQxZDcxNGEtOWZmYy00NjAxLTlhNjItZTI0MjdkY2EwMTRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI4VDA0NTQzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4YzlkNmRmMDkyOGY4YTdjMjk3NDFmNmJkNDE2YmQ4ZmNjOTQyZjZhMGQ2OWQyMDMyMGYwODdkNjlkMWZhM2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.m5zNCViN5ZpzFzDRsnCjFM1Jh5yDGthqvQz_mfEnR04 ?

Reply:
I can see it, but it is a `private-user-image`. Add the image to the repository and show it from there.

Reply:
how about https://github-production-user-asset-6210df.s3.amazonaws.com/39142679/286099389-97a77043-86af-4384-8612-65b2b81fa360.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20231128%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231128T050841Z&X-Amz-Expires=300&X-Amz-Signature=c671eb99090860ef5570ad6bff98001169623c662e51dd6a11ef271f43aa30bb&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=442118790 ?
I can access from secret browser

Reply:
replaced with local svg
89c0b9ccc2345ff70c79189c8fe8a27511953ca8